docs/triggers/link-triggers.md

```diff
@@ -0,0 +1,120 @@
+
+| Parameter name  | type     | Description                                                          |
+| ----------------|:-------------:| ---------------------------------------------------------------------|
+| `data.user_id`        | string            | A unique identifier for the Slack user who invoked the Trigger         |
+| `data.channel_id`        | string            | A unique identifier for the channel where the Trigger was invoked         |
+| `data.interactivity`        | object            | See [Block Kit interactivity](https://api.dev.slack.com/future/triggers/future/block-events)|
+| `data.location`        | string            | Where the Trigger was invoked. Can be `message` or `bookmark`|
+| `data.message_ts`        | string            | A unique UNIX timestamp in seconds indicating when the Trigger-invoking message was sent|
+| `data.user`        | object            | An object containing a `user_id` and a `secret` that can be used to identify and validate the specific user who invoked the Trigger|
+| `data.action_id`        | string            | A unique identifier for the action that invoked the Trigger. See [Block Kit interactivity](https://api.dev.slack.com/future/triggers/future/block-events) |
+| `data.block_id`        | string            | A unique identifier for the block where the Trigger was invoked. See [Block Kit interactivity](https://api.dev.slack.com/future/triggers/future/block-events)|
+| `data.bookmark_id`        | string            | A unique identifier for the bookmark where the Trigger was invoked|
+
+The data context can be used in the input parameter as follows:
+
+```ts
+{ 
+ inputs: {
+  a_input_value: {
+    value: "{{data.user_id}}"
+  }
+ }
+}
+```
+## Usage
+Below are some usage examples of `ShortcutTrigger` objects which can be used in a .ts file to create a `shortcut` trigger through the [Hermes CLI](./trigger-basics.md/#creating-triggers-using-the-hermes-cli), alternatively this object could be passed into a 
+`client.workflows.triggers.create` method to achieve the same effect at [runtime](./trigger-basics.md/#creating-triggers-in-the-runtime-environment).
+
+### Example Configured Shortcut Trigger
+
+```ts
+const trigger: ShortcutTrigger = {
+  type: "shortcut",
+  name: "Request Time off",
+  description: "Starts the workflow to request time off",
+  workflow: "#/workflows/reverse_workflow",
+  inputs: {
+    interactivity: {
+      value: "{{data.interactivity}}",
+    },
+  },
+  shortcut: {
+    button_text: "Click Me"
+  }
+};
+```
+
+### Example Shortcut Trigger Without Shortcut Object
+
+```ts
+const trigger: ShortcutTrigger = {
+  type: "shortcut",
+  name: "Request Time off",
+  description: "Starts the workflow to request time off",
+  workflow: "#/workflows/reverse_workflow",
+  inputs: {
+    interactivity: {
+      value: "{{data.interactivity}}",
+    },
+  },
+};
+```
+
+### Example Response Payload from Create API
```

Comment:
Lets make sure the response isn't surfacing any potentially sensitive data that could be used elsewhere